app/controllers/games_controller.rb

```diff
@@ -45,4 +56,26 @@ def load_game_graph_data
       { name: 'Challenges Solved', data: SolvedChallenge.group_by_hour(:created_at).count }
     ]
   end
+
+  def load_game_with_users
+    @game = Game.includes(:users).instance
+  end
+
+  private
+
+  # Creates a zip from any collection of files available on the user model.
+  # For example, create_zip_of('resume') will create a zip of all resumes
+  # uploaded by all teams, broken out into the team folders.
+  def create_zip_of(uploader)
+    compressed_filestream = Zip::OutputStream.write_buffer do |zos|
+      @game.users.each do |user|
+        next unless (file_contents = user.send(uploader).read)
```

Comment:
Does this handle the case of a user not having either document uploaded?

Reply:
Yes